packages/edit-site/src/components/site-hub/index.js

```diff
@@ -68,7 +68,7 @@ const SiteHub = memo(
 							label={ __( 'Go to the Dashboard' ) }
 							className="edit-site-layout__view-mode-toggle"
 							style={ {
-								transform: 'scale(0.5)',
+								transform: 'scale(0.5333) translateX(-4px)', // Offset to position the icon 12px from viewport edge
```

Comment:
I guess this is the trouble you're referring to. It's a curious number. How does the math work out?

Reply:
Yes. The icon/button is 60x60 in the editor and gets scaled down when you exit the editor. The `0.5` scale transform results in a 30x30 icon, which doesn't align with standard button sizes and creates awkward alignments. Transforming by `0.5333` creates a `32px` icon which is easier to align with other elements.

Reply:
Makes sense. I wonder if this needs to be in a comment?